No default JAVA_HOME

Without setting `JAVA_HOME` getting this error:

```
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.951 s
[INFO] Finished at: 2026-03-30T07:27:01Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project parser: Fatal error compiling: error: release version 11 not supported -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -26,6 +26,7 @@ ENV IMAGE_VERSION=$IMAGE_VERSION
 ENV PIP_BREAK_SYSTEM_PACKAGES 1
 ENV JAVA_17_HOME /opt/java/openjdk17
 ENV JAVA_21_HOME /opt/java/openjdk21
+ENV JAVA_HOME $JAVA_17_HOME
 
 # renovate: datasource=maven depName=org.owasp:dependency-check-maven versioning=maven
 ARG MAVEN_OWASP_DEPENDENCY_CHECK_PLUGIN_VERSION="8.3.1"

--- a/test/shell/tools.bats
+++ b/test/shell/tools.bats
@@ -19,3 +19,13 @@
   run java -version
   [ "$status" -eq 0 ]
 }
+
+@test "should have JAVA_HOME set to JDK 17" {
+  [ -n "$JAVA_HOME" ]
+  [[ "$JAVA_HOME" == *"openjdk17"* ]]
+}
+
+@test "should have JAVA_HOME pointing to valid JDK" {
+  run "$JAVA_HOME/bin/java" -version
+  [ "$status" -eq 0 ]
+}

